src/EnergyPlus/Coils/CoilCoolingDXCurveFitPerformance.cc

```diff
@@ -676,3 +677,13 @@ void CoilCoolingDXCurveFitPerformance::setOperMode(EnergyPlus::EnergyPlusData &s
                               this->object_name));
     }
 }
+void CoilCoolingDXCurveFitPerformance::oneTimeAvailSchedSetup()
+{
+    if (this->myOneTimeAvailSchedInitFlag) {
+        // set avail schedule for each mode
+        this->normalMode.coilCoolingDXAvailSched = static_cast<EnergyPlus::CoilCoolingDXPerformanceBase *>(this)->coilCoolingDXAvailSched;
```

Comment:
My IDE is telling me this `static_cast` isn't necessary. You might double check it's needed.